cluster-autoscaler/processors/provreq/injector.go

```diff
@@ -87,29 +97,32 @@ func (p *ProvisioningRequestPodsInjector) GetPodsFromNextRequest(
 	if err != nil {
 		return nil, err
 	}
-
 	for _, pr := range provReqs {
 		if !isSupportedClass(pr) {
+			klog.Warningf("Provisioning Class %s is not supported for ProvReq %s/%s", pr.Spec.ProvisioningClassName, pr.Namespace, pr.Name)
```

Comment:
I don't think we want to log the warning here. This method will be used in batch processing of check-capacity requests with a custom `isSupportedClass` function, and we'd end up logging it for best-effort-atomic-scale-up requests.

Can we move logging this to the `isSupportedClass` function defined in `Process()` instead?

Reply:
I can't anchor comment there but what I mean is:

```
		func(pr *provreqwrapper.ProvisioningRequest) bool {
			_, found := provisioningrequest.SupportedProvisioningClasses[pr.Spec.ProvisioningClassName]
			if !found {
				klog.Warningf("Provisioning Class %s is not supported for ProvReq %s/%s", pr.Spec.ProvisioningClassName, pr.Namespace, pr.Name)
			}
			return found
		})
```

Reply:
Done